services/core-web/src/components/mine/Permit/MinePermitTable.tsx

```diff
@@ -354,6 +344,7 @@ const columns = [
           cancelText="Cancel"
         >
           <Button ghost type="primary" size="small">
+            {/* @ts-ignore */}
```

Comment:
I'm just curious what all the ts-ignore flags on this file are for. Mostly I want to know if it's something we'll be able to resolve in the future or a rule that should be changed.

Reply:
I think most of these can be removed. I committed a change for this now.  For the button type I think typescript detects that the "secondary" value for type is not valid and we would get an error like this without the ts-ignore:

`TS2322: Type '"secondary"' is not assignable to type '"link" | "text" | "default" | "ghost" | "primary" | "dashed"'.`

I wasn't too sure if I should just remove the type attribute for this. Any thoughts?